Bottom: 2025ec45ba735bb4294012a57027f63f403b2709
Top:    7f0cd59597a88f71ef6b78cc8c57dcbaf661232d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-26 16:18:39 +0100

Refresh of fix-outline-drawing-to-only-ap

---

diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index 57d2b58..0a74759 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -844,11 +844,13 @@ ps_set_layer (const char *name, int group, int empty)
   /* If we're printing a copper layer other than the outline layer,
      and we want to "print outlines", and we have an outline layer,
      print the outline layer on this layer also.  */
-  if (global.outline
-      && global.outline_layer != NULL
-      && global.outline_layer != PCB->Data->Layer+idx
-      && strcmp (name, "outline")
-      && strcmp (name, "route"))
+  if (global.outline &&
+      global.is_copper &&
+      global.outline_layer != NULL &&
+      global.outline_layer != PCB->Data->Layer+idx &&
+      strcmp (name, "outline") &&
+      strcmp (name, "route")
+      )
     {
       dapi->draw_layer (global.outline_layer, &global.region, NULL);
     }
